services/server/server.go

```diff
@@ -279,15 +283,23 @@ func (s *Server) CheckUpdates(ctx context.Context, req *serverpb.CheckUpdatesReq
 		UpdateAvailable: v.UpdateAvailable,
 		LatestNewsUrl:   v.LatestNewsURL,
 	}
+
+	if updatesDisabled {
+		res.UpdateAvailable = false
```

Comment:
@BupycHuk basically, we will only replace `UpdateAvailable` with something like `CanUpdate` on that line.

Reply:
Yeah, I see that. So it's ok that we return latest version in response, right?

Reply:
Yes, client depends (should depend) only on `UpdateAvailable` value and should not compare version strings.
Please check that, btw.

Reply:
Checked frontend side and yes it uses `UpdateAvailable`, but alt+click will run update even if `updateAvailable` is false.

Reply:
That's okay, we protect from that in StartUpdate.

Reply:
LGTM then